src/applications/personalization/view-dependents/util/index.js

```diff
@@ -8,3 +11,25 @@ export async function getData(apiRoute, options) {
     return error;
   }
 }
+
+export function splitPersons(persons) {
+  const onAwardPeeps = [];
+  const notOnAwardPeeps = [];
+  const allPersons = {};
+
+  persons.map(person => {
```

Comment:
I think you could also use `.forEach` here, since the array returned from this `.map` isn't technically being used.